pkg/abi/linux/fuse.go

```diff
@@ -543,6 +647,32 @@ type FUSECreateMeta struct {
 	_     uint32
 }
 
+// FUSERenameIn sent by the kernel for FUSE_RENAME
+//
+// +marshal dynamic
+type FUSERenameIn struct {
+	Newdir primitive.Uint64
+	Oldname CString
+	Newname CString
+}
+
+// MarshalBytes implements marshal.Marshallable.MarshalBytes.
+func (r *FUSERenameIn) MarshalBytes(dst []byte) []byte {
+	dst = r.Newdir.MarshalBytes(dst)
+	dst = r.Oldname.MarshalBytes(dst)
+	return r.Newname.MarshalBytes(dst)
+}
+
+// UnmarshalBytes implements marshal.Marshallable.UnmarshalBytes.
+func (r *FUSERenameIn) UnmarshalBytes(buf []byte) []byte {
+	panic("Unimplemented, FUSERmDirIn is never unmarshalled")
+}
+
+// SizeBytes implements marshal.Marshallable.SizeBytes.
+func (r *FUSERenameIn) SizeBytes() int {
+	return 8 + r.Oldname.SizeBytes() + r.Newname.SizeBytes()
```

Comment:
Replace `8` with `r.Newdir.SizeBytes()` so that if one day `NewDir` type changes, we don't need to update this function.